internal/provider/kubernetes/config/envoy-gateway/kustomization.yaml

```diff
@@ -7,4 +7,4 @@ generatorOptions:
 configMapGenerator:
 - name: envoy-gateway-config
   files:
-  - envoy-gateway.yaml
+  - config.yaml
```

Comment:
We now have a config file mounted in a config directory. I thought the common pattern is the name of the config, e.g. envoy-gateway in the config directory.

Reply:
sure can rename config.yaml to envoy-gateway.yaml if thats the preference